src/FMI2/convert.jl

```diff
@@ -26,7 +26,7 @@ end
 export fmi2StatusToString
 
 """
-ToDo.
+converts a [`fmi2Causality`](@ref) to the corresponding String ("parameter", "calculatedParameter", "input", "output", "local", "independent")
```

Comment:
please use docstring format as defined in the [julialang doc](https://docs.julialang.org/en/v1/manual/documentation/)

Reply:
fixed